casper/src/test/resources/PoSTest.rho

```diff
@@ -97,15 +97,23 @@ match (
 
 
           contract test_close_block_finishes(rhoSpec, _, ackCh) = {
-            new resultCh in {
+            new resultCh, activeValidatorsCh, setBlockData(`rho:test:block:data:set`), blockDataSet in {
               @PoS!("closeBlock", *resultCh) |
               for ( _ <- resultCh) {
-                rhoSpec!("assertMany",
-                  [
-                    (true, "all good here")
-                  ],
-                  *ackCh
-                )
+                // After the first "closeBlock" is done, there are active validators
+                // For testing we need one of the active validators to be the block sender
+                @PoS!("getActiveValidators", *activeValidatorsCh) |
+                for (@activeValidators <- activeValidatorsCh) {
+                  setBlockData!("sender", activeValidators.nth(0), *blockDataSet) |
```

Comment:
For the other tests to work, we have to set the block sender to one of the active validators.
Since we potentially pick a new set of active validators in `PoS!closeBlock`, we have to set the sender after running `PoS!closeBlock`.

I just realized that we should do that after each call to `PoS!closeBlock`. So I guess I should create a helper contract that does that and replace every occurrence of `@PoS!("closeBlock", *closeBlockCh)` in `PoSTest.rho` with the helper contract.